specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -482,10 +482,8 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/ValidationToken"
+          "202": {
+            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously. Please poll custom domain {customDomainName} for the new validation token values."
```

Comment:
Do we need {customDomainName} in the description?

Reply:
Updated description to "Accepted. The request has been accepted for processing and the operation will complete asynchronously. Please poll the targeted custom domain and check the 'validationProperties' field for the new validation token values."